### DISCUSSION TITLE
How to use ghidra

### DISCUSSION THREAD

**Discussion by kai-ion** [NationalSecurityAgency/ghidra#4702] [Q&A] [answered]
I am new to ghidra. After I decompile a file I see a whole bunch of code. Is there a way to extract them all out of ghidra?

**Comment by seantomecko** [comment, marked as answer]
`File`->`Export Program`, then select `C/C++` from the dropdown menu to get decompiled C out of Ghidra, but you'll have to re-export if you annotate/rename any functions, xrefs, etc in order to get your changes.

**Reply by kai-ion** [reply]
Thank you!!
